starter_code/elevator.js

```diff
@@ -1,19 +1,51 @@
+const Person = require('./person.js');
+
 class Elevator {
   constructor(){
     this.floor      = 0;
     this.MAXFLOOR   = 10;
     this.requests   = [];
+    this.waitingList = [];
+    this.passengers = [];
+  }
+
+  start() {
+    setInterval(update, 1000);
```

Comment:
este update debería ser this.update ya que es un método de la clase que estás definiendo